tailwind.config.js

```diff
@@ -203,20 +197,5 @@ module.exports = {
       },
     },
   },
-  variants: {
-    extend: {
-      backgroundColor: ['checked', 'disabled'],
-      borderColor: ['checked', 'disabled'],
```

Comment:
Why remove the variants? My understanding is these are used for components of the redesign (but not 100% sure).

Reply:
With the JIT feature, all variants are enabled out of the box: https://v2.tailwindcss.com/docs/just-in-time-mode

> Every variant is enabled out of the box. Variants like `focus-visible`, `active`, `disabled`, and others are not normally enabled by default due to file-size considerations. Since this library generates styles on demand, you can use any variant you want, whenever you want. You can even stack them like `sm:hover:active:disabled:opacity-75`. Never configure your variants again.